crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/MaatCourtDataService.java

```diff
@@ -26,11 +28,16 @@ public class MaatCourtDataService {
     private final RestAPIClient maatAPIClient;
     private final ServicesConfiguration configuration;
 
-    public Contribution findContribution(Integer repId, String laaTransactionId) {
-        Contribution response = maatAPIClient.get(
-                new ParameterizedTypeReference<Contribution>() {},
+    public List<Contribution> findContribution(Integer repId, String laaTransactionId, boolean findLatestContribution) {
+
+        final MultiValueMap<String, Boolean> queryParams = new LinkedMultiValueMap<>();
+        queryParams.add("findLatestContribution", findLatestContribution);
+
+        List<Contribution> response = maatAPIClient.get(
```

Comment:
If there is no contributions then we get ErrorDTO from the mattAPIClient get call. Do we handle this scenario?